data/schema/schema_v1.json

```diff
@@ -533,16 +534,19 @@
                               "type": "integer"
                             },
                             "alias": {
-                              "type": "string"
+                              "type": "string",
+                              "description": "Allows another name for a variable to be used."
                             },
                             "repeats": {
-                              "type": "boolean"
+                              "type": "boolean",
+                              "description": "Value showing whether there are multiple answers for a question or not."
```

Comment:
I don't think repeats is actually used. It only appears in `test_navigation_confirmation.json`. If it's not used it should be removed from `test_navigation_confirmation.json` and this schema.

Reply:
Removed these after testing test_navigation_confirmation without the repeats element